packages/ddp-client/common/livedata_connection.js

```diff
@@ -1721,17 +1721,17 @@ export class Connection {
 
     self.onReconnect && self.onReconnect();
     DDP._reconnectHook.each(callback => {
-      callback(self);
+      callback(self, !self.shouldRetryMethods && self.errorNoRetry, oldOutstandingMethodBlocks);
       return true;
     });
 
     if (!self.shouldRetryMethods) {
-        self.shouldRetryMethods = true;
-        // Run the callback on all remaining method calls with the error as the parameter
-        while (oldOutstandingMethodBlocks.length) {
-            const methods = oldOutstandingMethodBlocks.pop().methods;
-            methods.forEach(m => m.receiveResult(self.errorNoRetry));
-        }
+      self.shouldRetryMethods = true;
+      // Run the callback on all remaining method calls with the error as the parameter
+      while (oldOutstandingMethodBlocks.length) {
+        const methods = oldOutstandingMethodBlocks.pop().methods;
+          methods.forEach(m => m.receiveResult(self.errorNoRetry));
```

Comment:
This whitespace change is in the wrong commit, also this line still has double-indent.

Reply:
fixed - moved to the right commit.

What do you mean by double-ident? It's on the same Indentation as the others from what I can see